yt_dlp/extractor/fathom.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime
+import time
+import json
+
+from .common import InfoExtractor
+from ..utils import (
+    get_element_html_by_id,
+    extract_attributes,
+)
```

Comment:
you can easily keep these sorted with `isort` (though we may be moving to hatch/ruff in the near future)
```suggestion
import json

from .common import InfoExtractor
from ..utils import (
    extract_attributes,
    float_or_none,
    get_element_html_by_id,
    parse_iso8601,
)
from ..utils.traversal import traverse_obj
```